pipelines/lib/util_test.groovy

```diff
@@ -1994,6 +1994,73 @@ def void runDispatchVerify(Map p) {
   } // withEnv
 } // runDispatchVerify
 
+/**
```

Comment:
Sorry about the duplication, but one goal was to allow testing of more substantial modifications to the Jenkins scripts without impacting normal pipeline execution; parameterization of the existing scripts was thought to be more dangerous.